### PR TITLE
Fixing Federal Christmas Shift.

### LINKED ISSUE #241: 2017 UnitedStates Christmas Day calculation is wrong
```
(datetime.date(2017, 12, 24), 'Christmas Eve'),
(datetime.date(2017, 12, 25), 'Christmas Day'),
(datetime.date(2017, 12, 25), 'Christmas Eve (Observed)'),
(datetime.date(2017, 12, 26), 'Christmas Eve (Observed)'),
(datetime.date(2017, 12, 27), 'Christmas Day (Observed)')
```

It should be:

```
(datetime.date(2017, 12, 24), 'Christmas Eve'),
(datetime.date(2017, 12, 25), 'Christmas Day'),
(datetime.date(2017, 12, 25), 'Christmas Eve (Observed)'),
(datetime.date(2017, 12, 26), 'Christmas Day (Observed)')
```

### PR DESCRIPTION
**Pull request by Natim**
Fixes #241
Refs #232

### CODE REVIEW

**Comment by Natim** [comment]
Tests are failing because of #245

**Comment by brunobord** [comment]
For some reasons, travis refuses to install the Python 3.3 interpreter, which causes those tests to fail.
Please rebase your branch from master in order to at least fix this one.

File: workalendar/usa/core.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class UnitedStates(WesternCalendar, ChristianMixin):
         (7, 4, 'Independence Day'),
     )
     # Veterans day label
+    include_veterans_day = True
```
Comment (brunobord) [inline workalendar/usa/core.py]: I can't see the point of this flag.
Veterans day is unanimously observed by all 50 States, there's no case where you'd have a "False" value here.

Comment (Natim) [reply workalendar/usa/core.py]: https://github.com/Natim/Diggory/blob/master/diggory/calendar/usa.py#L9

**Pull request merged**